sql/2019/14_CMS/14_10.sql

```diff
@@ -0,0 +1,33 @@
+#standardSQL
+# 14_10: CrUX FCP performance of CMS providers
+SELECT
+  app,
+  COUNT(DISTINCT origin) AS freq,
+  IF(form_factor.name = 'desktop', 'desktop', 'mobile') AS form_factor,
```

Comment:
Great question. Tablet is included here because when the HTTP Archive websites are grouped into desktop or mobile we do the same exact `IF(form_factor.name = 'desktop', 'desktop', 'mobile')` segmentation of the CrUX origins. (The HA websites are sourced from the CrUX dataset itself.) So when joining the two together, we're following the same convention.

The Performance chapter did it a little differently because the datasets are never joined, it's based entirely in the CrUX data.

Hopefully consistent enough not to confuse readers.